engine/src/main/java/io/zeebe/engine/processing/bpmn/behavior/BpmnStateTransitionBehavior.java

```diff
@@ -164,6 +164,23 @@ private void verifyIncidentResolving(final BpmnElementContext context, final Str
     }
   }
 
+  public <T extends ExecutableFlowNode>
+      BpmnElementContext transitionToCompletedWithParentNotification(
```

Comment:
I'm wondering whether this method should just be merged into transitionToCompleted, but let's keep it like this for now.